src/Tribe/Main.php

```diff
@@ -260,7 +260,7 @@ public function load_assets() {
 			[
 				[ 'tribe-ui', 'tribe-ui.css', [ 'tec-variables-full' ] ],
 				[ 'tribe-buttonset', 'buttonset.js', [ 'jquery', 'underscore' ] ],
-				[ 'tribe-common-admin', 'tribe-common-admin.css', [ 'tec-variables-skeleton', 'tec-variables-full', 'tribe-dependency-style', 'tribe-bumpdown-css', 'tribe-buttonset-style', 'tribe-select2-css' ] ],
+				[ 'tribe-common-admin', 'tribe-common-admin.css', [ 'tec-variables-skeleton', 'tec-variables-full', 'tribe-dependency-style', 'tribe-bumpdown-css', 'tribe-buttonset-style', 'tribe-select2-css', 'datatables-css' ] ],
```

Comment:
@EugeneKyale @Camwyn we shouldn't add this here, it's loading the `datatables-css` into everyone of our pages, when we actually only need it on the EA one. The correct place would be to add it to the dependencies of the following asset: `tribe-ea-styles`

Reply:
@bordoni ooh okay, I'll make the correction and open a new pr.

Reply:
@bordoni I've addressed your comments in the following 2 PRs.
- https://github.com/the-events-calendar/tribe-common/pull/1882
- https://github.com/the-events-calendar/the-events-calendar/pull/4144